ansible/roles/icinga/defaults/main.yml

```diff
@@ -371,13 +403,14 @@ icinga__director_register_host_vars: {}
 icinga__director_register_host_object:
   object_type: 'object'
   object_name: '{{ icinga__fqdn }}'
-  address: '{{ icinga__fqdn }}'
-  imports: '{{ lookup("flattened",
-               (icinga__director_register_default_templates
-                + icinga__director_register_templates
-                + icinga__director_register_group_templates
-                + icinga__director_register_host_templates),
-               wantlist=True) }}'
+  display_name: '{{ icinga__display_name }}'
+  address: '{{ ansible_default_ipv4.address | d(ansible_all_ipv4_addresses | d([]) | first) | d(icinga_fqdn) }}'
+  address6: '{{ ansible_default_ipv6.address | d(ansible_all_ipv6_addresses | d([]) | first) | d(omit, true) }}'
```

Comment:
It would be great if you could make the `address` and `address6` parameters configurable as well. My production servers have their DNS names configured here, and I would like to keep it that way because it would be one less thing to worry about during a network renumbering.

Reply:
Sure, will do